es/13_3_Potenciando_Bitcoin_con_Scripts.md

```diff
@@ -0,0 +1,317 @@
+
+En esta situación, podría publicar virtuosamente la transacción, cerrando el canal proto-Lightning. Ella pone la transacción en la cadena y espera 24 horas. Bob no puede hacer nada al respecto porque no tiene el código de recuperación. Luego, después de la espera, Alice reclama sus fondos. (Bob hace lo mismo con su propia transacción de compromiso final).
+
+### Bloqueo con contratos codificados con bloqueo de tiempo
+
+Las Transacciones de Compromiso Revocable fueron solo un trampolín hacia Lightning. La Lightning Network real utiliza un mecanismo más complejo llamado [contrato de bloqueo de tiempo hash] (https://en.bitcoin.it/wiki/Hashed_Timelock_Contracts), o HTLC.
+
+El objetivo principal de los HTLC es crear una red integral de participantes. Las transacciones ya no son solo entre un par de participantes que han ingresado a la red juntos, sino que ahora pueden ser entre personas previamente no asociadas. Cuando se envían fondos, se crea una serie de transacciones, cada una de ellas bloqueada con un "secretHash". Cuando se revela el "secretCode" correspondiente, se puede gastar toda la cadena de transacciones. Esto es lo que permite que las transacciones singulares se conviertan realmente en una red.
+
+También hay un poco más de complejidad en los scripts de bloqueo de Lightning Network. Hay bloqueos separados para el remitente y el destinatario de cada transacción que son más divergentes que las diferentes transacciones de compromiso mencionadas en la sección anterior. Vamos a mostrarlos a ambos para demostrar el poder de estos scripts de bloqueo, pero no vamos a detenernos en cómo interactúan entre sí.
+
+#### Bloquear la transacción del destinatario
+
+Una vez más, comenzaremos a ver la transacción de compromiso de Alice, que muestra los fondos que recibió:
+
+```
+OP_HASH160 
+OP_DUP 
+<secretHash> 
+OP_EQUAL
+    
+IF
+
+    <+24Hours>
+    OP_CHECKSEQUENCEVERIFY
+    OP_2DROP
+    <pubKeyAlice>
+    
+ELSE
+
+    <revokeHash> 
+    OP_EQUAL
+        
+    OP_NOTIF
+            
+        <Date> 
+        OP_CHECKLOCKTIMEVERIFY 
+        OP_DROP
+        
+    ENDIF
+        
+    <pubKeyBob>
+    
+ENDIF
+    
+OP_CHECKSIG
+```
+
+La clave de estos nuevos HTLC es el `secretHash`, que dijimos es lo que permite que una transacción abarque la red. Cuando la transacción se ha extendido desde su originador hasta su destinatario previsto, se revela el "secretCode", que permite a todos los participantes crear un "secretHash" y desbloquear toda la red de pagos.
+
+Una vez que se ha revelado el `secretCode`, se abre la sucursal` IF`: Alice puede reclamar los fondos 24 horas después de que la transacción se haya realizado en la red Bitcoin.
+
+Sin embargo, también existe la oportunidad de que Bob reclame sus fondos, que aparecen en la rama "ELSE". Puede hacerlo si la transacción ha sido revocada (pero Alice la pone en la cadena de bloques de todos modos), _o si_ ha ocurrido un tiempo de espera absoluto.
```

Comment:
- **Replace** double quotes (` " `) in `"ELSE"` **by** backticks (`` ` ``).
- `_o si_ ha finalizado un tiempo de espera absoluto.`